### PR TITLE
Update behavior ingest (varying delay period) & new tracking ingest directory format

### PR DESCRIPTION
**Pull request by ttngu207**

### CODE REVIEW

File: pipeline/tracking.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ class TrackingDevice(dj.Lookup):
     """
     contents = [
        ('Camera 0', 'side', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)'),
-       ('Camera 1', 'bottom', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)')]
+       ('Camera 1', 'bottom', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)'),
+        ('Camera 2', 'body', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)')]
```
Comment (LiuDaveLiu) [inline pipeline/tracking.py]: Camera 2 frame rate is 100 Hz, or 1/0.01

File: pipeline/tracking.py [hunk 2]
```diff
@@ -23,7 +23,8 @@ class TrackingDevice(dj.Lookup):
     """
     contents = [
        ('Camera 0', 'side', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)'),
-       ('Camera 1', 'bottom', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)')]
+       ('Camera 1', 'bottom', 1/0.0034, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)'),
+        ('Camera 2', 'body', 1/0.01, 'Chameleon3 CM3-U3-13Y3M-CS (FLIR)')]
```
Comment (ixcat) [inline pipeline/tracking.py]: will this trigger indentation problems?

File: pipeline/ingest/behavior.py [hunk 3]
```diff
@@ -713,11 +611,6 @@ def make(self, key):
             rows['trial_event'], ignore_extra_fields=True,
             allow_direct_insert=True, skip_duplicates=True)
 
-        log.info('BehaviorIngest.make(): ... CorrectedTrialEvents')
-        BehaviorIngest().CorrectedTrialEvents().insert(
```
Comment (ixcat) [inline pipeline/ingest/behavior.py]: this + related values were intended to log any adjustments made to the raw input data due to bad gui duration items - we are not using the gui durations now but calculating, but are the 'end' cases true?

File: pipeline/tracking.py (hunk 2 above)
Comment (LiuDaveLiu) [reply pipeline/tracking.py]: I doubt it, I don't think we are doing any calculations on the tracking data yet.

File: pipeline/ingest/behavior.py (hunk 3 above)
Comment (LiuDaveLiu) [reply pipeline/ingest/behavior.py]: I am unsure what you mean, which 'end' cases?

Comment (ixcat) [reply pipeline/ingest/behavior.py]: there are a few blocks removed which added things to 'CorrectedTrialEvents' which are now removed (so nothing is logged to corrected trial events) - I don't recall the specific source cases but it was something about trials without a duration at the 'ends' of a session taking a duration value from previous trials or something like that (in retrospect, should have noted better in the code - that said, git history should have details).

Basically the question is: do things still need to be noted in CorrectedTrialEvents

Comment (ttngu207) [reply pipeline/ingest/behavior.py]: ah, yeah the `CorrectedTrialEvents` I believe was there to notate which event didn't have the right "duration" read from the GUI. Now that we're not using GUI for duration anymore, this becomes obsolete thus removed
